GooglePay/src/main/java/com/braintreepayments/api/GooglePayRequest.java

```diff
@@ -196,7 +196,7 @@ public void setGoogleMerchantName(@Nullable String merchantName) {
     }
 
     public void setEnvironment(@Nullable String environment) {
-        this.environment = "PRODUCTION".equals(environment.toUpperCase()) ? "PRODUCTION" : "TEST";
+        this.environment = "PRODUCTION".equalsIgnoreCase(environment) ? "PRODUCTION" : "TEST";
```

Comment:
We should create a separate PR for these Google Pay changes.

Reply:
whoops looks like auto-formatting got it